lib/gc/Transforms/MemRefToCPURuntime.cpp

```diff
@@ -38,9 +40,38 @@ bool hasParallelParent(Operation *op) {
   }
   return false;
 }
-struct AlignedAllocLowering : public OpRewritePattern<memref::AllocOp> {
-  using OpRewritePattern<memref::AllocOp>::OpRewritePattern;
-  LogicalResult matchAndRewrite(memref::AllocOp op,
+
+uint64_t getMemRefSizeInBytes(MemRefType memrefType) {
+  if (ShapedType::isDynamicShape(memrefType.getShape()))
```

Comment:
I guess we should use our allocator for dynamic shaped tensor by default?

Reply:
Yes, if the memref is dynamic, then `getMemRefSizeInBytes` will return `UINT64_MAX`, shapes larger than `128Bytes` will be converted to gc runtime allocator.

Reply:
Sorry. You are right.